packages/vue/src/components/FInteractiveTable/FInteractiveTable.vue

```diff
@@ -275,13 +275,30 @@ export default defineComponent({
             required: false,
             default: undefined,
         },
+        /**
+         * Enable showing the active row.
+         */
+        showActive: {
+            type: Boolean,
+            required: false,
+            default: true,
```

Comment:
Optimalt skulle jag nog säga att denna borde vara opt-in men det vore ju en brytande ändring så vi kör så här.